sphinx/templates/latex/longtable.tex_t

```diff
@@ -8,7 +8,7 @@
 <%- endif -%>
 <%= table.get_colspec() %>
 <%- if table.caption -%>
-\caption{<%= ''.join(table.caption) %>}<%= labels %>\\
+\caption{<%= ''.join(table.caption) %>\strut}<%= labels %>\\*[\sphinxafterLTcaptionskip]
```

Comment:
Other macros for longtable is named as "...longtable..." with full spelling. But this is abbreviated.
Is there any rule? IMO, it would be better to use same naming rule.